spec/datadog/ci/configuration/settings_spec.rb

```diff
@@ -18,6 +18,10 @@
     end
 
     describe '#ci' do
+      after do
+        settings.reset!
+      end
+
```

Comment:
I don't think this is needed? `subject(:settings)` above seems to be defined as a new object that gets recreated for every testcase, so there shouldn't be any leftover state between tests that needs resetting.

Also, the tests still pass for me if I remove this :)